code/modules/sanity/breakdown.dm

```diff
@@ -38,6 +38,7 @@
 	holder.owner << img
 	flick(icon_state, img)
 	holder.owner.playsound_local(get_turf(holder.owner), breakdown_sound, 100)
+	spawn(100)
```

Comment:
Why this?

Reply:
Owo waht is this?

Reply:
Reere wants people to have some time window before some breakdown like hysteria would turn them into combat-incapable bodies. I hope that ten seconds after eye animation appeared is enough.

Reply:
Ten seconds is hardly enough, we need at least 50, plus they should receive a clear warning that they about to breakdown and collapse.

Reply:
Excuse me wtf?! There is one breakdown out of 17 that can disable you during combat. 10 seconds is more than enough to disengage from any situation where you aren't fucked up and tbh if you don't control your sanity - it's your fault. But there is even more important thing: there is a Stalwart and a Lesson Learnt breakdowns that can save your ass in combat and if the latter has non-combat uses then Stalwart becomes 99% useless if it would trigger a minute after you got your ass kicked.

Reply:
Sanity breakdowns should not disable you during combat at all. All of the players who experienced it reported that it's the most frustrating thing, and frustration is something that hurts the combat tremendously in any game. So it does not matter what positive breakdowns do here, it's about that.

So we need to provide players with enough time to end or exit the combat, along with clear warning about that.